[{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28893424","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28893424","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28893424,"user":{"login":"moscicki","id":621965,"avatar_url":"https://gravatar.com/avatar/56282d408e288b61469589f1c98f4f86?d=https%3a%2f%2fidenticons.github.com%2fa73f312232a793606ee9b2c0615df705.png&r=x","gravatar_id":"56282d408e288b61469589f1c98f4f86","url":"https://api.github.com/users/moscicki","html_url":"https://github.com/moscicki","followers_url":"https://api.github.com/users/moscicki/followers","following_url":"https://api.github.com/users/moscicki/following{/other_user}","gists_url":"https://api.github.com/users/moscicki/gists{/gist_id}","starred_url":"https://api.github.com/users/moscicki/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/moscicki/subscriptions","organizations_url":"https://api.github.com/users/moscicki/orgs","repos_url":"https://api.github.com/users/moscicki/repos","events_url":"https://api.github.com/users/moscicki/events{/privacy}","received_events_url":"https://api.github.com/users/moscicki/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t14:33:28z","updated_at":"2013-11-20t14:33:28z","body":"if i rename the file (in the same local folder) the problem persists.\r\n\r\nfor smaller files in the same local folder on the client (5mb, 50mb) i don't have the sync problem.\r\n\r\n"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28894720","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28894720","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28894720,"user":{"login":"deepdiver1975","id":1005065,"avatar_url":"https://gravatar.com/avatar/bdba071ed68da9ce8edae53c364902ef?d=https%3a%2f%2fidenticons.github.com%2f67e101ad9531b7d25697b76e503e062e.png&r=x","gravatar_id":"bdba071ed68da9ce8edae53c364902ef","url":"https://api.github.com/users/deepdiver1975","html_url":"https://github.com/deepdiver1975","followers_url":"https://api.github.com/users/deepdiver1975/followers","following_url":"https://api.github.com/users/deepdiver1975/following{/other_user}","gists_url":"https://api.github.com/users/deepdiver1975/gists{/gist_id}","starred_url":"https://api.github.com/users/deepdiver1975/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/deepdiver1975/subscriptions","organizations_url":"https://api.github.com/users/deepdiver1975/orgs","repos_url":"https://api.github.com/users/deepdiver1975/repos","events_url":"https://api.github.com/users/deepdiver1975/events{/privacy}","received_events_url":"https://api.github.com/users/deepdiver1975/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t14:49:01z","updated_at":"2013-11-20t14:49:01z","body":"@moscicki we have seen a similar case a few days back.\r\n\r\ncan i ask you to attach the access log? we need to know which http status case has been sent back for that put.\r\nany error in the web server logs? thx\r\n\r\n@danimo @dragotin fyi\r\n\r\nlet's track this issue in here for now until we know if it's a server or client bug"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28895256","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28895256","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28895256,"user":{"login":"moscicki","id":621965,"avatar_url":"https://gravatar.com/avatar/56282d408e288b61469589f1c98f4f86?d=https%3a%2f%2fidenticons.github.com%2fa73f312232a793606ee9b2c0615df705.png&r=x","gravatar_id":"56282d408e288b61469589f1c98f4f86","url":"https://api.github.com/users/moscicki","html_url":"https://github.com/moscicki","followers_url":"https://api.github.com/users/moscicki/followers","following_url":"https://api.github.com/users/moscicki/following{/other_user}","gists_url":"https://api.github.com/users/moscicki/gists{/gist_id}","starred_url":"https://api.github.com/users/moscicki/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/moscicki/subscriptions","organizations_url":"https://api.github.com/users/moscicki/orgs","repos_url":"https://api.github.com/users/moscicki/repos","events_url":"https://api.github.com/users/moscicki/events{/privacy}","received_events_url":"https://api.github.com/users/moscicki/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t14:55:03z","updated_at":"2013-11-20t14:55:03z","body":"hello,\r\n\r\nthank you for very quick response.\r\n\r\nthe status is 201 in all cases.\r\n\r\nhere is the failed chunk upload:\r\n\r\n137.138.32.188 - moscicki [20/nov/2013:15:50:38 +0100] \"get /owncloud/status.php http/1.1\" 200 87\r\n137.138.32.188 - - [20/nov/2013:15:50:39 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/movie-2.3gb-chunking-3367763637-220-0 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:51:04 +0100] \"post /owncloud/index.php/heartbeat http/1.1\" 200 -\r\n\r\nhere are successful smaller files uploads:\r\n\r\n137.138.32.188 - - [20/nov/2013:15:29:01 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/b.5mb http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:02 +0100] \"proppatch /owncloud/remote.php/webdav/blah/chep-video/b.5mb http/1.1\" 207 305\r\n137.138.32.188 - - [20/nov/2013:15:29:02 +0100] \"propfind /owncloud/remote.php/webdav/blah/chep-video/b.5mb http/1.1\" 207 459\r\n\r\n137.138.32.188 - - [20/nov/2013:15:29:02 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb-chunking-3342317964-5-0 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:03 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb-chunking-3342317964-5-1 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:05 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb-chunking-3342317964-5-2 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:05 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb-chunking-3342317964-5-3 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:06 +0100] \"put /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb-chunking-3342317964-5-4 http/1.1\" 201 -\r\n137.138.32.188 - - [20/nov/2013:15:29:07 +0100] \"proppatch /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb http/1.1\" 207 316\r\n137.138.32.188 - - [20/nov/2013:15:29:07 +0100] \"propfind /owncloud/remote.php/webdav/blah/chep-video/sample.file.50mb http/1.1\" 207 471\r\n\r\nkuba\r\n\r\n--\r\n\r\non nov 20, 2013, at 3:49 pm, thomas mã¼ller <notifications@github.com> wrote:\r\n\r\n> @moscicki we have seen a similar case a few days back.\r\n> \r\n> can i ask you to attach the access log? we need to know which http status case has been sent back for that put.\r\n> any error in the web server logs? thx\r\n> \r\n> @danimo @dragotin fyi\r\n> \r\n> let's track this issue in here for now until we know if it's a server or client bug\r\n> \r\n> â€”\r\n> reply to this email directly or view it on github.\r\n> "},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28905191","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28905191","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28905191,"user":{"login":"dragotin","id":1070214,"avatar_url":"https://gravatar.com/avatar/af829234b31fb915704c1c65ef48eee0?d=https%3a%2f%2fidenticons.github.com%2f3392afab7f424e10623ba95a941f4d34.png&r=x","gravatar_id":"af829234b31fb915704c1c65ef48eee0","url":"https://api.github.com/users/dragotin","html_url":"https://github.com/dragotin","followers_url":"https://api.github.com/users/dragotin/followers","following_url":"https://api.github.com/users/dragotin/following{/other_user}","gists_url":"https://api.github.com/users/dragotin/gists{/gist_id}","starred_url":"https://api.github.com/users/dragotin/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/dragotin/subscriptions","organizations_url":"https://api.github.com/users/dragotin/orgs","repos_url":"https://api.github.com/users/dragotin/repos","events_url":"https://api.github.com/users/dragotin/events{/privacy}","received_events_url":"https://api.github.com/users/dragotin/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t16:39:33z","updated_at":"2013-11-20t16:39:33z","body":"hmm, maybe the php version can not handle files larger than > 1.1 gb? thats a natural border for older php versions on some distros."},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28906942","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28906942","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28906942,"user":{"login":"moscicki","id":621965,"avatar_url":"https://gravatar.com/avatar/56282d408e288b61469589f1c98f4f86?d=https%3a%2f%2fidenticons.github.com%2fa73f312232a793606ee9b2c0615df705.png&r=x","gravatar_id":"56282d408e288b61469589f1c98f4f86","url":"https://api.github.com/users/moscicki","html_url":"https://github.com/moscicki","followers_url":"https://api.github.com/users/moscicki/followers","following_url":"https://api.github.com/users/moscicki/following{/other_user}","gists_url":"https://api.github.com/users/moscicki/gists{/gist_id}","starred_url":"https://api.github.com/users/moscicki/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/moscicki/subscriptions","organizations_url":"https://api.github.com/users/moscicki/orgs","repos_url":"https://api.github.com/users/moscicki/repos","events_url":"https://api.github.com/users/moscicki/events{/privacy}","received_events_url":"https://api.github.com/users/moscicki/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t16:57:43z","updated_at":"2013-11-20t16:57:43z","body":"i have php 5.4.16 --> i am not sure how this may be related to the file size -> the upload is chunked. so the physical files created are only 10mb and we only get to create the first chunk and never to the stage when we concatenate the chunks.\r\n\r\nunless there is a total file size passed around some function calls in the owncloud php framework and some limit kicks in there. but then on a similar server running 5.0.11 community with php 5.3.3 i don't see this problem.\r\n\r\n\r\n\r\non nov 20, 2013, at 5:39 pm, dragotin <notifications@github.com> wrote:\r\n\r\n> hmm, maybe the php version can not handle files larger than > 1.1 gb? thats a natural border for older php versions on some distros.\r\n> \r\n> â€”\r\n> reply to this email directly or view it on github.\r\n> "},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28908875","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28908875","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28908875,"user":{"login":"dragotin","id":1070214,"avatar_url":"https://gravatar.com/avatar/af829234b31fb915704c1c65ef48eee0?d=https%3a%2f%2fidenticons.github.com%2f3392afab7f424e10623ba95a941f4d34.png&r=x","gravatar_id":"af829234b31fb915704c1c65ef48eee0","url":"https://api.github.com/users/dragotin","html_url":"https://github.com/dragotin","followers_url":"https://api.github.com/users/dragotin/followers","following_url":"https://api.github.com/users/dragotin/following{/other_user}","gists_url":"https://api.github.com/users/dragotin/gists{/gist_id}","starred_url":"https://api.github.com/users/dragotin/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/dragotin/subscriptions","organizations_url":"https://api.github.com/users/dragotin/orgs","repos_url":"https://api.github.com/users/dragotin/repos","events_url":"https://api.github.com/users/dragotin/events{/privacy}","received_events_url":"https://api.github.com/users/dragotin/received_events","type":"user","site_admin":false},"created_at":"2013-11-20t17:18:10z","updated_at":"2013-11-20t17:18:10z","body":"yes, the upload is chunked, but once all parts are uploaded the original file is assembled again. can you find the final file in the servers file system?"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28966630","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28966630","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28966630,"user":{"login":"moscicki","id":621965,"avatar_url":"https://gravatar.com/avatar/56282d408e288b61469589f1c98f4f86?d=https%3a%2f%2fidenticons.github.com%2fa73f312232a793606ee9b2c0615df705.png&r=x","gravatar_id":"56282d408e288b61469589f1c98f4f86","url":"https://api.github.com/users/moscicki","html_url":"https://github.com/moscicki","followers_url":"https://api.github.com/users/moscicki/followers","following_url":"https://api.github.com/users/moscicki/following{/other_user}","gists_url":"https://api.github.com/users/moscicki/gists{/gist_id}","starred_url":"https://api.github.com/users/moscicki/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/moscicki/subscriptions","organizations_url":"https://api.github.com/users/moscicki/orgs","repos_url":"https://api.github.com/users/moscicki/repos","events_url":"https://api.github.com/users/moscicki/events{/privacy}","received_events_url":"https://api.github.com/users/moscicki/received_events","type":"user","site_admin":false},"created_at":"2013-11-21t08:53:58z","updated_at":"2013-11-21t08:53:58z","body":"no, actually the only file related to this upload that i see is the first chunk in /cache... i don't see anything in /files"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/28968169","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-28968169","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":28968169,"user":{"login":"deepdiver1975","id":1005065,"avatar_url":"https://gravatar.com/avatar/bdba071ed68da9ce8edae53c364902ef?d=https%3a%2f%2fidenticons.github.com%2f67e101ad9531b7d25697b76e503e062e.png&r=x","gravatar_id":"bdba071ed68da9ce8edae53c364902ef","url":"https://api.github.com/users/deepdiver1975","html_url":"https://github.com/deepdiver1975","followers_url":"https://api.github.com/users/deepdiver1975/followers","following_url":"https://api.github.com/users/deepdiver1975/following{/other_user}","gists_url":"https://api.github.com/users/deepdiver1975/gists{/gist_id}","starred_url":"https://api.github.com/users/deepdiver1975/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/deepdiver1975/subscriptions","organizations_url":"https://api.github.com/users/deepdiver1975/orgs","repos_url":"https://api.github.com/users/deepdiver1975/repos","events_url":"https://api.github.com/users/deepdiver1975/events{/privacy}","received_events_url":"https://api.github.com/users/deepdiver1975/received_events","type":"user","site_admin":false},"created_at":"2013-11-21t09:20:50z","updated_at":"2013-11-21t09:20:50z","body":"@dragotin i was think at first about a timeout on reassembly - but as @moscicki pointed out it's already the first chunk which is logged with that error code 5"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/29509475","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-29509475","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":29509475,"user":{"login":"karlitschek","id":867445,"avatar_url":"https://gravatar.com/avatar/87ce2b4531ee1a0c32b50e0d8f049224?d=https%3a%2f%2fidenticons.github.com%2f06e7098636caebfc9c30c9f52eb905df.png&r=x","gravatar_id":"87ce2b4531ee1a0c32b50e0d8f049224","url":"https://api.github.com/users/karlitschek","html_url":"https://github.com/karlitschek","followers_url":"https://api.github.com/users/karlitschek/followers","following_url":"https://api.github.com/users/karlitschek/following{/other_user}","gists_url":"https://api.github.com/users/karlitschek/gists{/gist_id}","starred_url":"https://api.github.com/users/karlitschek/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/karlitschek/subscriptions","organizations_url":"https://api.github.com/users/karlitschek/orgs","repos_url":"https://api.github.com/users/karlitschek/repos","events_url":"https://api.github.com/users/karlitschek/events{/privacy}","received_events_url":"https://api.github.com/users/karlitschek/received_events","type":"user","site_admin":false},"created_at":"2013-11-29t11:00:52z","updated_at":"2013-11-29t11:00:52z","body":"any news here?"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/29510945","html_url":"https://github.com/owncloud/core/issues/5964#issuecomment-29510945","issue_url":"https://api.github.com/repos/owncloud/core/issues/5964","id":29510945,"user":{"login":"dragotin","id":1070214,"avatar_url":"https://gravatar.com/avatar/af829234b31fb915704c1c65ef48eee0?d=https%3a%2f%2fidenticons.github.com%2f3392afab7f424e10623ba95a941f4d34.png&r=x","gravatar_id":"af829234b31fb915704c1c65ef48eee0","url":"https://api.github.com/users/dragotin","html_url":"https://github.com/dragotin","followers_url":"https://api.github.com/users/dragotin/followers","following_url":"https://api.github.com/users/dragotin/following{/other_user}","gists_url":"https://api.github.com/users/dragotin/gists{/gist_id}","starred_url":"https://api.github.com/users/dragotin/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/dragotin/subscriptions","organizations_url":"https://api.github.com/users/dragotin/orgs","repos_url":"https://api.github.com/users/dragotin/repos","events_url":"https://api.github.com/users/dragotin/events{/privacy}","received_events_url":"https://api.github.com/users/dragotin/received_events","type":"user","site_admin":false},"created_at":"2013-11-29t11:31:10z","updated_at":"2013-11-29t11:31:10z","body":"@moscicki for test reasons, could you add another couple of bytes to the failing file in question and see if it still fails? maybe it's any kind of off-by-one problem with the httpbf lib? not sure..."}]